seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumJSEnv.scala

```diff
@@ -1,25 +1,19 @@
 package org.scalajs.jsenv.selenium
 
-import org.openqa.selenium._
-import org.openqa.selenium.remote.DesiredCapabilities
-import org.openqa.selenium.remote.server._
 
 import org.scalajs.jsenv._
 
 import java.net.URL
 import java.nio.file.{Path, Paths}
+import org.openqa.selenium.Capabilities
+import org.openqa.selenium.remote.DesiredCapabilities
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.JavascriptExecutor
+import org.openqa.selenium.Platform
 
-final class SeleniumJSEnv(capabilities: Capabilities, config: SeleniumJSEnv.Config) extends JSEnv {
-  def this(capabilities: Capabilities) =
-    this(capabilities, SeleniumJSEnv.Config())
+final class SeleniumJSEnv(config: SeleniumJSEnv.Config) extends JSEnv {
```

Comment:
```suggestion
final class SeleniumJSEnv(driverFactory: () => Driver, config: SeleniumJSEnv.Config) extends JSEnv {
```

And remove it from the config? Feels like this is clearer.

Reply:
Agreed, especially that it's essentially a required parameter.

Reply:
Agreed, especially that it's essentially a required parameter.